test/resources/mnist/model_cpu/code/mnist.py

```diff
@@ -16,6 +16,11 @@
 import os
 import sys
 
+try:
+    import transformers
+except ImportError:
+    raise ImportError("The 'transformers' module was not found.'")
```

Comment:
Curious why this explicit try-catch is necessary here.

Reply:
Acknowledged. Removed try-catch block. If `transformers` is not present, the error will show up anyways. Added comment above `import` statement to explain the reason for importing `transformers` i.e. to check that dependencies in `requirements.txt` get installed.